image/scanner/rhel/konflux.Dockerfile

```
@@ -45,7 +45,7 @@ LABEL \
     summary="The image scanner for Red Hat Advanced Cluster Security for Kubernetes" \
     url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c" \
     # We must set version label to prevent inheriting value set in the base stage.
-    version="${SCANNER_TAG}" \
+    version="${SCANNER_TAG}-${TARGETARCH}" \
```

I think, this won't harm but isn't necessary. `version` label is just for cosmetics. It has to be there, it has to carry some meaning for a human that's in the know, but that's it. Arch could be derived from other image attributes (there's `architecture` label that seems to be auto-populated).
```suggestion
    version="${SCANNER_TAG}" \
```

and besides `TARGETARCH` isn't declared as `ARG` that's passed to buildah.

```json
            "Labels": {
                "architecture": "x86_64",
                "build-date": "2024-07-26T09:35:20",
                "com.redhat.component": "rhacs-scanner-container",
                ...
                "version": "2.34.x-29-g982071d17e-fast-"
            }
```